packages/apron/apron.20150930/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "1.2"
+authors: ["Bertrant Jeannet" "Antoine Miné"]
+homepage: "http://apron.cri.ensmp.fr/library/"
+maintainer: "Nicolas Berthier <m@nberth.space>"
+# dev-repo: "svn+ssh://scm.gforge.inria.fr/svnroot/apron/apron/trunk"
+bug-reports: "https://gforge.inria.fr/tracker/?atid=8946&group_id=2625&func=browse"
+license: "LGPL-2.1 with linking exception"
+build: [
+  ["sh" "./configure" "--prefix" "%{apron:share}%"
+  		      "--no-ppl" {! conf-ppl:installed}
+		      "--no-java"
+		      "--absolute-dylibs" { os = "darwin" } ]
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "apron"]
+  ["rm" "-r" "-f" "%{apron:share}%"]
+]
+tags: [ "flags:light-uninstall" ]
+depends: [
+  "ocamlfind"
+  "camlidl"
+  "mlgmpidl"
+]
+depopts: [
+  "conf-ppl"
+]
+available: [
+  opam-version >= "1.2"
```

Comment:
That's garbage from older versions of opam specifications for this package, from a time when repositories needed to work with earlier version of opam. Now that's probably useless indeed, so I'll remove that.